redis_prometheus_exporter/resource.tf

```diff
@@ -1,14 +1,24 @@
+data cloudfoundry_space local_space {
```

Comment:
why local_space and local_instance? should it be redis_space and redis_instance?